docs/admin/pool_operations.md

```diff
@@ -906,17 +971,34 @@ Target (rank 5 idx 1) is down.
 These should be the same values used when reintegrating the targets.
 
 ```
-$ dmg pool reintegrate $DAOS_POOL --rank=5 --target-idx=0,1
+$ dmg pool reintegrate $DAOS_POOL --ranks=5 --target-idx=0,1
 ```
 
 !!! warning
     While dmg pool query and list show how many targets are disabled for each
     pool, there is currently no way to list the targets that have actually
     been disabled. As a result, it is recommended for now to try to reintegrate
     all engine ranks one after the other via `for i in seq $NR_RANKs; do dmg
-    pool reintegrate --rank=$i; done`. This limitation will be addressed in the
+    pool reintegrate --ranks=$i; done`. This limitation will be addressed in the
     next release.
```

Comment:
shouldn't we rewrite the for loop to construct a list of all ranks and pass it to one dmg pool reintegrate --ranks invocation a s a list (now that this is supported), instead of calling the reintegrate separately for each rank?

Reply:
I think @johannlombardi is doing another pass over this area of the documentation soon, so rather than change the backport to be out of sync with the master commit maybe he can make this update then?

Reply:
This warning is confusing! Surely you don't want to go through every rank and instead you only want to go through the list of disabled ranks reported by pool query?